source/documentation/virtual_machine_management_guide/topics/Blacklisting_Network_Interfaces.adoc

```diff
@@ -1,12 +1,12 @@
 :_content-type: PROCEDURE
 [id="Blacklisting_Network_Interfaces"]
-= Blacklisting Network Interfaces
+= Ignore Network Interfaces
 
-You can configure the `ovirt-guest-agent` on a virtual machine to ignore certain NICs. This prevents IP addresses associated with network interfaces created by certain software from appearing in reports. You must specify the name and number of the network interface you want to blacklist (for example, `eth0`, `docker0`).
+You can configure the `ovirt-guest-agent` on a virtual machine to ignore certain NICs. This prevents IP addresses associated with network interfaces created by certain software from appearing in reports. You must specify the name and number of the network interface you want to ignore (for example, `eth0`, `docker0`).
 // Note that once https://bugzilla.redhat.com/show_bug.cgi?id=1546675 is resolved, this procedure can be simplified (it won't require configuring the blacklist before first start of the agent)
 [IMPORTANT]
 ====
-You must blacklist NICs on the virtual machine before the guest agent is started for the first time.
+You must ignore NICs on the virtual machine before the guest agent is started for the first time.
```

Comment:
This sounds like the user is ignoring the NICs. 

Whenever you see a comment that mentions a bug, check whether the bug has been resolved. In this case, BZ1546675 is resolved, so you should remove this note.